x-pack/legacy/plugins/apm/public/components/app/ServiceMap/index.tsx

```diff
@@ -5,18 +5,51 @@
  */
 
 import theme from '@elastic/eui/dist/eui_theme_light.json';
-import React from 'react';
-import { useFetcher } from '../../../hooks/useFetcher';
+import React, { useMemo, useEffect, useState, useRef } from 'react';
+import { find } from 'lodash';
+import { i18n } from '@kbn/i18n';
+import { EuiButton } from '@elastic/eui';
+import { toMountPoint } from '../../../../../../../../src/plugins/kibana_react/public';
+import { ServiceMapAPIResponse } from '../../../../server/lib/service_map/get_service_map';
+import {
+  Connection,
+  ServiceConnectionNode,
+  ConnectionNode
+} from '../../../../common/service_map';
 import { useLicense } from '../../../hooks/useLicense';
 import { useUrlParams } from '../../../hooks/useUrlParams';
 import { Controls } from './Controls';
 import { Cytoscape } from './Cytoscape';
 import { PlatinumLicensePrompt } from './PlatinumLicensePrompt';
+import { useCallApmApi } from '../../../hooks/useCallApmApi';
+import { useDeepObjectIdentity } from '../../../hooks/useDeepObjectIdentity';
+import { getAPMHref } from '../../shared/Links/apm/APMLink';
+import { useLocation } from '../../../hooks/useLocation';
+import { LoadingOverlay } from './LoadingOverlay';
+import { useApmPluginContext } from '../../../hooks/useApmPluginContext';
 
 interface ServiceMapProps {
   serviceName?: string;
 }
 
+type Element =
```

Comment:
Not sure this is necessary with the built-in types in https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/cytoscape/index.d.ts.